### PR TITLE
use relative path

### PR DESCRIPTION
**Pull request by theTd**

### CODE REVIEW

File: assets/js/site.js [hunk 1]
```diff
@@ -262,6 +262,7 @@ function printPort(port) {
 
 $(document).ready(function() {
 	var socket = io.connect({
+        path: window.location.pathname + 'socket.io',
```
Comment (Cryptkeeper) [inline assets/js/site.js]: Does pathname always end with a / regardless of the request? Otherwise this could malform.

Comment (hugmanrique) [inline assets/js/site.js]: Most of the times the loaded page will be the index.html, but most people just use the slash to access it.

According to the MDN docs, a slash isn't guaranteed.

**Comment by theTd** [comment]
I'm not sure what you means as I'm not pro on it. How to make it works in all situations?


 ------------------原始邮件 ------------------发件人：Hugo Manrique <notifications@github.com>日期：2017-08-12 19:01:01收件人：Cryptkeeper/Minetrack <Minetrack@noreply.github.com>抄送：靛 <i@thetd.me>, Author <author@noreply.github.com>主题：Re: [Cryptkeeper/Minetrack] use relative path (#88)@hugmanrique commented on this pull request.



In assets/js/site.js:
> @@ -262,6 +262,7 @@ function printPort(port) {
 
 $(document).ready(function() {
 	var socket = io.connect({
+        path: window.location.pathname + 'socket.io',

Most of the times the loaded page will be the index.html, but most people just use the slash to access it.
According to the MDN docs, a slash isn't guaranteed.

**Comment by theTd** [comment]
does the problem solved?

**Pull request closed without merging**